Analysis/library.py

```diff
@@ -282,35 +281,37 @@ def binning(temp_x, objno, bin_pts_input, interp_file, mname = '', fitspath0 = '
         The mass array saved here contains the 4140 masses (log(mass)) with the no mass indices replaced
         with interpolated values; it does NOT exclude sources excluded for binning (indices relative to
         original table). 
-        The luminosity array saved here contains the 4140 luminosities (log(lum)) with the >44 and nans replaced
-        with -1.
+        The luminosity array saved here contains the 4140 luminosities (log(lum)) with the >44 and nans 
+        replaced with -1.
         '''
-        np.savez(out_file, mass = np.log10(temp_x), lum = lum, bin_ind = bin_ind, bin_start = bin_start, logM_min = bin_edge, 
-                 logM_max = bin_redge, flux = flux, wavelength = wavelength, logM_avg = mass_avg, logM_median = mass_median,
-                 bin_ID = bin_ID, N_stack = N, logLHb_min = lowest_hbeta, logLHb_max = highest_hbeta, logLHb_avg = lum_avg, 
+        np.savez(out_file, mass = np.log10(temp_x), lum = lum, bin_ind = bin_ind, bin_start = bin_start, 
+                 logM_min = bin_edge, logM_max = bin_redge, flux = flux, wavelength = wavelength, 
+                 logM_avg = mass_avg, logM_median = mass_median, bin_ID = bin_ID, N_stack = N, 
+                 logLHb_min = lowest_hbeta, logLHb_max = highest_hbeta, logLHb_avg = lum_avg, 
                  logLHb_median = lhb_median)
         
-        if adaptive == False:
+        if not adaptive:
             out_ascii = fitspath0 + str(bin_pts_input) + '_binning.tbl' 
         else:
             out_ascii = fitspath0 + filename_dict['bin_info']
-        if hbeta_bin == False:
+        if not hbeta_bin:
             lowest_hbeta = np.zeros(len(bin_pts_input))
             highest_hbeta = np.zeros(len(bin_pts_input))
             lum_avg = np.zeros(len(bin_pts_input))
             lhb_median = np.zeros(len(bin_pts_input))
         column_names0 = remove_from_list(bin_names0, [bin_names0[-1]])
         all_column_names = tuple(merge_column_names(column_names0, bin_mzevolve_names0))
-        table_stack = Table([bin_ID, N, bin_edge, bin_redge, mass_avg, mass_median, lowest_hbeta, highest_hbeta, lum_avg, lhb_median], names = all_column_names)        
+        table_stack = Table([bin_ID, N, bin_edge, bin_redge, mass_avg, mass_median, lowest_hbeta, 
+                             highest_hbeta, lum_avg, lhb_median], names = all_column_names)        
```

Comment:
If you do use a `dict` above, you can drop elements and pass it into Table, simplifying things.